examples/EmailerAgent/emailer/agent.py

```diff
@@ -121,40 +121,10 @@ def __init__(self, config_path, **kwargs):
         # Keep track of keys that have been added to send with.
         self.tosend = {}
 
-        def onstart(sender, **kwargs):
-            self.vip.pubsub.subscribe('pubsub', topics.PLATFORM_SEND_EMAIL,
-                                      self.on_email_message)
-
-            self.vip.pubsub.subscribe('pubsub', topics.ALERTS_BASE,
-                                      self.on_alert_message)
-            self.vip.pubsub.subscribe('pubsub',
-                                      prefix=topics.ALERTS.format(agent_class='',
-                                                                  agent_uuid=''),
-                                      callback=self.on_alert_message)
-
-        self.core.onstart.connect(onstart, self)
-        self.sent_alert_emails = defaultdict(int)
-
-#    @Core.periodic(10)
-#    def accessService(self):
-#       headers = {
-#           "something": 'something'
-           #"to-addresses": ['alpha.beta@foo.com', 'bob-and-joe@bar.com']
-#       }
-
-#       message = {
-#           "subject": "Something is not right",
-#           "message": "Somethign is down"
-#       }
-
-       self.vip.pubsub.publish('pubsub', topic='platform/send_email',headers=headers ,message=message)
-
-
     def _test_smtp_address(self, smtp_address,smtp_port,smtp_username,smtp_password):
         try:
             server = smtplib.SMTP(self.current_config.get('smtp_address', None),self.current_config.get('smtp_port', None))
```

Comment:
I just saw this...does this require the tls here for the testing smtp address?  must not